src/tsday/dto/create-tsday.dto.ts

```diff
@@ -17,6 +17,10 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsDate, IsInt, IsNotEmpty, IsString, Max, Min } from 'class-validator';
 
+const MIN_MINS = 0;
+const MAX_MINS = 0;
```

Comment:
I think max mins should be 600 (10 hours) or 720 (12 hours). I'm thinking 12 hours is probably better here

Reply:
I will one day remember mins > hrs. Change made